### PR TITLE
i made changes to the german version.

### PR DESCRIPTION
**Pull request by fronx**
it may or may not be slightly better than @mortice's. :]
(i'm natively german.)

### CODE REVIEW

**Comment by janl** [comment]
:+1:

**Comment by fronx** [comment]
@mortice i prefer 'jemand' because it sounds less formal because it's shorter, and because it doesn't have a gender-specific ending.

**Comment by rentalcustard** [comment]
:+1: for non-gender-specificity!

**Comment by ntlk** [comment]
shall I merge in

**Comment by ntlk** [comment]
because I don't understand so much german

**Comment by ntlk** [comment]
Also maybe someone should ask @tef for commit rights so someone with knowledge of German can handle the Berlin pull requests

**Comment by fronx** [comment]
@tef may i have commit rights, pleeeease? :grinning:

**Comment by ntlk** [comment]
Can someone just check I didn't screw up, there was a million and one merge conflicts (???)

**Comment by ghost** [comment]
@mortice to clarify: Both versions are correct but "zu jemandem" is nicer German.

http://www.belleslettres.eu/artikel/jemand-niemand-deklination.php

File: de_berlin.html [hunk 1]
```diff
@@ -38,82 +38,81 @@
...
-      <p><i>Wenn du Angst davor hast, nicht Computer genug zu sein, dann komm.</i></p>
+      <p><i>Wenn du Angst davor hast, nicht ausreichend Computer zu sein, dann komm.</i></p>
 
       <p>If your day job isn't code, come.</p>
-      <p><i>Wenn du nicht beruflich Computer bist, dann komm.</i></p>
+      <p><i>Wenn deine Arbeit nicht Code ist, dann komm.</i></p>
 
       <p>If you think you're an imposter, come.</p>
-      <p><i>Wenn du glaubst, dass du ein Imposter bist, dann komm.</i></p>
+      <p><i>Wenn du glaubst, du bist ein Imposter, dann komm.</i></p>
```
Comment (igalic) [inline de_berlin.html:65]: Warum nicht imposter übersetzen? Ich glaube Hochstapler ist die treffendste übersetzung hier.

File: de_berlin.html [hunk 2]
```diff
@@ -18,19 +18,19 @@
...
-      <p><i>Da du jetzt verstehst, lassma Computer anonym gehen.</i></p>
+      <p><i>Jetzt verstehst du. Also lass mal zu Computer Anonym gehen.</i></p>
 
       <h4>The Plan</h4>
-      <h4><i>Das Plan</i></h4>
+      <h4><i>Der Plan</i></h4>
 
       <p>This might be the group for you if you want to meet socially conscious nerds, to talk about interesting things: This is not an enterprenuerial meetup, nor is it networking. It is a support group.</p>
-      <p><i>Wenn du Bock drauf hast, sozial kompetente Computerfreaks zu treffen, um über interessante Sachen zu reden, vielleicht ist die Gruppe für dich geiegnet: Die ist wider ein Unternehmerstammtisch, noch Networking. Die ist eine Selbsthilfegruppe.</i></p>
+      <p><i>Diese Gruppe könnte richtig für dich sein, wenn du Lust hast, Nerds mit sozialer Ader zu treffen, um über interessante Sachen zu reden. Es ist kein Unternehmerstammtisch, noch Networking. Es ist eine Selbsthilfegruppe.</i></p>
```
Comment (igalic) [inline de_berlin.html:27]: entweder:

```
Es ist weder ein Unternehmerstammtisch, noch Networking.
```

oder

```
Es ist weder Unternehmerstammtisch, noch Networking.
```

Gemischt schaut's nicht so toll aus.

File: de_berlin.html [hunk 3]
```diff
@@ -38,82 +38,81 @@
...
-      <p><i><a href="http://geekfeminism.wikia.com/wiki/Conference_anti-harassment/Policy">Ein standarder Verhaltenscodex gilt</a>. Das heißt: Nicht Leute mobben. Keine nachteilige Witze machen, selbst nicht "ironisch".</i></p>
+      <p><i><a href="http://geekfeminism.wikia.com/wiki/Conference_anti-harassment/Policy">Ein Standard-Verhaltenscodex gilt</a>. Das bedeutet: Du darfst keinen mobben. Du darfst keine Witze machen, die andere ausgrenzen. Nicht einmal "ironisch".</i></p>
 
       <ul>
         <li>- We want to be inclusive; we don't welcome homophobic, racist, transphobic, or sexist behavior, amongst others.</li>
-        <li><i>- Wir wollen inklusiv sein; wir bewerten u.a. homosexualitätsfeindliches, racistisches, transphobisches oder sexistisches Verhalten negativ.</i></li>
+        <li><i>- Wir wollen inklusiv sein; wir bewerten u.a. homophobes, rassistisches, transphobes und sexistisches Verhalten negativ.</i></li>
         <li>- We think feminism is a good thing; we think brogramming is a bad thing.</li>
         <li><i>- Wir halten Feminismus für gut; wir halten Brogramming für schlecht.</i></li>
```
Comment (igalic) [inline de_berlin.html:84]: Die Zeile ist schon länger geändert worden.

**Pull request merged**